services/services.go

```diff
@@ -407,3 +411,95 @@ func (opts namespaceOpts) SetMetadataNamespace(v string) NamespaceOptions {
 	opts.metadata = v
 	return opts
 }
+
+type placementOptions struct {
```

Comment:
Why do we need to move the options here? Feels to me it's better to keep the implementation (and perhaps even the interface) under `services/placement`?

Reply:
PlacementOptions is already exposed in services/types.go so I'd rather keep the options impl in the services/ package since it's tightly tied to the interface.

Reply:
I'm not sure if that's the right approach. Feels to me the options implementation is tightly coupled with the placement itself it makes sense to leave them inside `placement`. Why not move the placement options interface to `services/placement`? After all they are placement options so it actually makes sense for them to live inside the `placement` package no?